### ISSUE TITLE
bug/Language options are not used in the ocr table.

### ISSUE THREAD

**Issue by Inkorak** [Unstructured-IO/unstructured#1694]
**Describe the bug**
If you try to extract tables from pdf scans using metadata.text_as_html, then the text in these tables is not correct if the language is not English, even if you sent an argument with the language for the ocr. As far as I looked, UnstructuredTableTransformerModel is used to get tables, for ocr it uses tesseract or paddle. [But at the same time, no arguments with language are sent to them.
](https://github.com/Unstructured-IO/unstructured-inference/blob/fd0cc54fe1eca53dd2f815ceb4b01d93b608e234/unstructured_inference/models/tables.py#L89)
**To Reproduce**
`elements = partition(file=f, skip_infer_table_types=[], include_page_breaks=True, pdf_infer_table_structure=True, ocr_languages="eng+rus", strategy='hi_res')`
Any PDF with tables that is not in English.

**Expected behavior**
The language argument was also used in the ocr in the UnstructuredTableTransformerModel.

**Screenshots**
![2023-10-10_13-14-43](https://github.com/Unstructured-IO/unstructured/assets/52286717/5457d1ff-a9c5-4394-a2fe-1a5a790519bb)
The language argument is not sent to the pytesseract.

**Comment by yuming-long** [comment]
Hi there!

It is already fixed in `0.10.25` when I close this same issue: https://github.com/Unstructured-IO/unstructured/issues/1564.

I also refactored the table OCR logic so the code from your screenshot is deprecated and was moved to `unstructured` repo. The table OCR data is coming from https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/partition/ocr.py#L407 for tesseract and it is using the `lang` parameter.
